lib/liveValidation/liveValidator.ts

```diff
@@ -215,7 +215,18 @@ export class LiveValidator {
     }
 
     this.logging("Apply global transforms for all specs");
-    this.loader.transformLoadedSpecs();
+    try {
+      this.loader.transformLoadedSpecs();
+    } catch (e) {
+      const errMsg = `Failed to transform loaded specs, detail error message:${e?.message}.ErrorStack:${e?.stack}`;
+      this.logging(
+        errMsg,
+        LiveValidatorLoggingLevels.error,
+        LiveValidatorLoggingTypes.error,
+        "Oav.liveValidator.initialize.transformLoadedSpecs"
```

Comment:
do you need to add the doc for this error code ?

Reply:
This is the function name output in the log.